packages/vm/tests/config.js

```diff
@@ -119,11 +119,17 @@ const SKIP_VM = [
  * @returns {String} Either an alias of the forkConfig param, or the forkConfig param itself
  */
 function getRequiredForkConfigAlias(forkConfig) {
+  // SpuriousDragon is named EIP158 (attention: misleading name)
+  // in the client-independent consensus test suite
+  if (String(forkConfig).match(/^spuriousDragon$/i)) {
+    return 'EIP158'
```

Comment:
Spurious Dragon is named EIP 158? I think it is Tangerine Whistle? https://blog.ethereum.org/2016/10/18/faq-upcoming-ethereum-hard-fork/

Reply:
These names are a total mess, but I also hope that I didn't mix things up here? Think the tests shouldn't otherwise work.  Let me see: the blog article you link is referencing the `TangerineWhistle` fork (checked by the block number stated in the article, 2463000) and in the article it is also stated that this fork is also referenced as `EIP-150`. So `EIP-158` for `SpuriousDragon` should be correct I guess? Can someone confirm?

Reply:
Yes - sorry, you are right. These names are indeed a mess. EIP 158 was superseded by EIP 161 which is included in Spurious Dragon. 

I find it extremely confusing though that this is named as EIP158 in the consensus test suite.

Can we explicitly check:

1) That we are actually implementing [EIP 161](https://eips.ethereum.org/EIPS/eip-161) here and NOT EIP 158? Because the name of these tests implies we are testing EIP 158, while we should test for EIP 161 in Spurious Dragon. See "rationale" of EIP 161 where this EIP differs from EIP 158.

2) Or, if EIP 158 in the tests is actually intended to be EIP 161?

Reply:
Really don't get confused by the name here, again - this is a mess - but these tests are just testing all of `SpuriousDragon`, and no specific EIP, that is just the setup of the consensus tests, you can't choose specific EIPs but test the whole HF.